lms/djangoapps/certificates/api.py

```diff
@@ -540,7 +540,12 @@ def _get_two_letter_language_code(language_code):
     Shortens language to only first two characters (e.g. es-419 becomes es)
     This is needed because Catalog returns locale language which is not always a 2 letter code.
     """
-    return language_code[:2] if language_code else None
+    if language_code is None:
+        return None
+    elif language_code == '':
+        return ''
```

Comment:
Whats the difference between None vs ' '?

Reply:
I'm not 100% sure on that. I believe `None` means that language was never set, wheras `''` means that it was explicitly set to 'All Languages'. There is a distinction between the two in the tests and it didn't look to me like the behavior described there matched the actual code. This change fixes that.